hr_utilization_analysis/report/hr_utilization_analysis.py

```diff
@@ -108,82 +166,19 @@ class HrUtilizationAnalysisEntry(models.TransientModel):
         related="employee_id.parent_id",
         store=True,
     )
-    date = fields.Date(string="Date", required=True)
+    date = fields.Date(required=True)
     line_ids = fields.Many2many(
-        string="Timesheet Lines",
-        comodel_name="account.analytic.line",
-        compute="_compute_line_ids",
-        store=True,
-    )
-    capacity = fields.Float(string="Capacity", compute="_compute_capacity", store=True)
-    amount = fields.Float(string="Quantity", compute="_compute_amount", store=True)
-    difference = fields.Float(
-        string="Difference", compute="_compute_difference", store=True,
+        string="Timesheet Lines", comodel_name="account.analytic.line",
     )
 
+    capacity = fields.Float()
+    amount = fields.Float(string="Quantity")
+    difference = fields.Float()
+
     _sql_constraints = [
         (
             "entry_uniq",
             "UNIQUE(analysis_id, employee_id, date)",
             "An analysis entry for employee/date pair has to be unique!",
         ),
     ]
-
-    @api.depends("employee_id", "date")
-    def _compute_line_ids(self):
-        AccountAnalyticLine = self.env["account.analytic.line"]
-
-        for entry in self:
-            entry.line_ids = AccountAnalyticLine.search(
-                [
-                    ("project_id", "!=", False),
-                    ("employee_id", "=", entry.employee_id.id),
-                    ("date", "=", entry.date),
-                ]
-            )
-
-    @api.depends("employee_id", "date")
-    def _compute_capacity(self):
-        Module = self.env["ir.module.module"]
-
-        project_timesheet_holidays = Module.with_user(SUPERUSER_ID).search(
-            [("name", "=", "project_timesheet_holidays"), ("state", "=", "installed")],
-            limit=1,
-        )
-
-        for entry in self:
-            tz = pytz.timezone(entry.employee_id.resource_calendar_id.tz)
-            from_datetime = datetime.combine(entry.date, time.min).replace(tzinfo=tz)
-            to_datetime = datetime.combine(entry.date, time.max).replace(tzinfo=tz)
-
-            capacity = entry.employee_id._get_work_days_data(
-                from_datetime,
-                to_datetime,
-                compute_leaves=not project_timesheet_holidays,
-            )["hours"]
```

Comment:
@alexey-pelykh I don't get this part.. why were you setting `compute_leaves` explicitly to False when `project_timesheet_holidays` is installed? My impression is when you leave it on True, it will take the leaves into account and the answer will always be correct?